Source/ProtocolBuffers/ProtoBufCodeGenApp/ProtoBufCodeGen.dpr

```diff
@@ -62,29 +64,50 @@ begin
   for I := 1 to L do
     begin
       S := ParamStr(I);
-      if StrMatchLeft(S, '--pas_out=', False) or
-         StrMatchLeft(S, '-O=', False) then
-        begin
-          Delete(S, 1, 10);
-          ParamOutputPath := S;
-        end
-      else
-      if StrMatchLeft(S, '--proto_path=', False) or
-         StrMatchLeft(S, '-I=', False) then
-        begin
-          Delete(S, 1, 13);
-          ParamProtoPath := S;
-        end
-      else
-      if StrEqualNoAsciiCase(S, '--help') then
-        begin
-          PrintHelp;
-          Halt(1);
-        end
+      if StrMatchLeft(S, '--pas_ver=', False) then
+      begin
+        Delete(S, 1, 10);
+        N := StrToIntDef(S, 0);
+        if (N >= Integer(Low(TCodeGenSupportVersion))) and (N <= Integer(High(TCodeGenSupportVersion))) then
+          ParamSupportVersion := TCodeGenSupportVersion(N);
```

Comment:
[dcc32 Error] ProtoBufCodeGen.dpr(76): E2003 Undeclared identifier: 'ParamSupportVersion'

Reply:
Se i changed flcProtoBufProtoCodeGenPascal.pas

Reply:
Did you make a new commit? Not seeing changes. The .dpr file needs to change to declare those new variables.

Reply:
Variables not declared in .dpr file, e.g.

var N : Integer;

Reply:
Thanks!